youtube_dl/options.py

```diff
@@ -520,7 +520,7 @@ def _comma_separated_values_options_callback(option, opt_str, value, parser):
         '--external-downloader',
         dest='external_downloader', metavar='COMMAND',
         help='Use the specified external downloader. '
-             'Currently supports %s' % ','.join(list_external_downloaders()))
+             '(currently supported: %s)' % ' '.join(list_external_downloaders()))
```

Comment:
As dstftw used to say, unrelated change.

Reply:
It is related to:

```python
        selections = re.findall("\\(currently supported: (.*)\\)", help)
```

We should use `(currently supported: (.*)` to get the completion list.

```shell
❯ youtube-dl --convert-subs <Tab>
selection
ass  lrc  srt  vtt
```

comes from

```python
    postproc.add_option(
        '--convert-subs', '--convert-subtitles',
        metavar='FORMAT', dest='convertsubtitles', default=None,
        help='Convert the subtitles to other format (currently supported: srt ass vtt lrc)')
```